### ISSUE TITLE
A11y: Self-collapsing mode - Buttons must have discernible text

### ISSUE THREAD

**Issue by shopwareBot** [shopware/platform#5041]
Ensure buttons have discernible text. [https://dequeuniversity.com/rules/axe/4.10/button-name?application=AxeChrome|https://dequeuniversity.com/rules/axe/4.10/button-name?application=AxeChrome|smart-link] 

{noformat} <button id="clear-button-option-id-01922ef4de887f9b84cf9b1be233e3c1" class="swag-customized-products-date-picker-button date-picker-clear me-3 pe-4" type="button" tabindex="-1">
 <button id="open-button-option-id-01922ef4de887f9b84cf9b1be233e3c1" class="swag-customized-products-date-picker-button date-picker-open" type="button" tabindex="-1">
 <button id="clear-button-option-id-01922ef4de887f9b84cf9b2afb1b959d" class="swag-customized-products-date-picker-button date-picker-clear me-3 pe-4" type="button" tabindex="-1">
 <button id="open-button-option-id-01922ef4de887f9b84cf9b2afb1b959d" class="swag-customized-products-date-picker-button date-picker-open" type="button" tabindex="-1">
 <button class="btn btn-sm">
 <button class="customized-products-upload-files-element__container-close-button" tabindex="-1">{noformat}

h3. To solve this problem, you need to fix at least (1) of the following:

* Element does not have inner text that is visible to screen readers
* aria-label attribute does not exist or is empty
* aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
* Element has no title attribute
* Element's default semantics were not overridden with role="none" or role="presentation"

!Screen Shot 2024-10-09 at 08.35.26 AM.png|width=894,height=390,alt="Screen Shot 2024-10-09 at 08.35.26 AM.png"!

!Screen Shot 2024-10-09 at 08.35.40 AM.png|width=894,height=394,alt="Screen Shot 2024-10-09 at 08.35.40 AM.png"!

!Screen Shot 2024-10-09 at 08.36.36 AM.png|width=920,height=440,alt="Screen Shot 2024-10-09 at 08.36.36 AM.png"!

!Screen Shot 2024-10-09 at 08.36.48 AM.png|width=894,height=448,alt="Screen Shot 2024-10-09 at 08.36.48 AM.png"!

!Screen Shot 2024-10-09 at 10.36.55 AM.png|width=1060,height=498,alt="Screen Shot 2024-10-09 at 10.36.55 AM.png"!

!Screen Shot 2024-10-09 at 11.57.25 AM.png|width=940,height=694,alt="Screen Shot 2024-10-09 at 11.57.25 AM.png"!

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **CUS-731**



[created from CUS-731, comment 501896]

**Comment by shopwareBot** [comment]
[public]

 > Status Change: **In Progress**

 Work on this issue has started. It is now being implemented.

 

[created from CUS-731, comment 511989]
